cmd/kubeadm/app/apis/kubeadm/types.go

```diff
@@ -101,4 +101,16 @@ type NodeConfiguration struct {
 	NodeName                 string
 	TLSBootstrapToken        string
 	Token                    string
+
+	// TLSDiscoveryRootCAPubKeys specifies a set of public key pins. The root CA
+	// discovered during TLS bootstrapping must match one of these values.
```

Comment:
Ah, yeah I'll fix that comment and see if I made the same mistake elsewhere. I originally approached this code without a clear understanding of the "discovery" vs. "TLS bootstrapping" terminology.